lib/serializer.js

```diff
@@ -5,16 +5,22 @@ const pb = require('protobufjs')
 
 module.exports.init = function (apiFile/* : string */) {
   return pb.load(apiFile || './lib/api.proto').then((r) => {
-    return new Serializer({
+    if (this.serializer) { return this.serializer }
```

Comment:
i see; i didn't intend multiple calls of init in the same module to be used outside of tests. the intended pattern is module calls init, saves the serializer in its own scope, calls the saved serializer in the future. 

not a big deal either way